### ISSUE TITLE
Probable Typo in desugar.arr

### ISSUE THREAD

**Issue by belph** [brownplt/pyret-lang#658]
[This line](https://github.com/brownplt/pyret-lang/blob/a5d99d0e9a1ace2e6f3504d62e2b913dd97949eb/src/arr/compiler/desugar.arr#L473) in `desugar.arr` refers to `A.s-ann` (where `A` is `ast.arr`), which does not seem to exist. I'm *guessing* this line was intended to be
```
  | s-ref(l, ann) => A.s-ref(l, desugar-ann(ann))
```

**Comment by blerner** [comment]
I guess we don't use `s-ref` very often, then, huh? ;-)  Good catch.

**Comment by sorawee** [comment]
I don't even think that it's possible to construct a `s-ref` from a source code...

**Comment by blerner** [comment]
Yup, it's not used anywhere, so far as I can tell.  @jpolitz, wanna just delete it altogether?

**Comment by blerner** [comment]
@jpolitz ping

**Comment by schanzer** [comment]
@jpolitz ping

**Comment by schanzer** [comment]
@jpolitz ping
